lib/sheldon.rb

```diff
@@ -0,0 +1,51 @@
+require 'fileutils'
+require 'pathname'
+require 'yaml/store'
+require_relative 'sheldon/helpers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.